src/styles/footer.scss

```diff
@@ -0,0 +1,18 @@
+h3 {
+  margin-bottom: 0;
+  margin-top: 0;
+}
+
+#footer {
+  height: 20px;
+  left: 696px;
+  margin-top: 0;
+  position: absolute;
+  top: 95.5%;
+  width: 302px;
+}

```

Comment:
We should avoid setting things in terms of pixels. For size such as width and height, try to use vw (or any other responsive unit). For absolute positioning, use percent or vw usually.

Try something like
```
bottom: 2vw;
left: 50%;
```

Reply:
Also @reginawang99 love the new footer but can we make it an svg? Preferably in white, black, and full color! Font sizes are often variable and leaving it as raw text can cause issues like the one Matt asked me to fix in the template tonight.

Reply:
you're the designer so you know best! we had an offline discussion regarding svgs + links + selectable text